client/src/Components/Recipes/Recipe.js

```diff
@@ -123,14 +186,16 @@ const styles = theme => ({
             deletedInstructionData: [],
             editedInstructionData: [],
             favoriteButtonText: "Add to Favourite"
+            ,
+            userFullName: ""
         }
     }
 
     componentDidMount() {
         this.state.recipe_id = this.getUrlVars()["recipe_id"]
         getUID().then(user => {
             this.setState({uid: user.uid})
-            fetch(util.format('%s/api/recipe?uid=%s&recipe_id=%s', process.env.REACT_APP_EXPRESS_BACKEND, user.uid, this.state.recipe_id), {
+            fetch(util.format('%s/api/dashboard?uid=%s&recipe_id=%s', process.env.REACT_APP_EXPRESS_BACKEND, user.uid, this.state.recipe_id), {
```

Comment:
As explained in the chat, these changes need to be reverted. The recipe endpoint should be reconfigured to give us the user's name